### ISSUE TITLE
Bug: Creating a statement before a typed variable causes a crash. 

### ISSUE THREAD

**Issue by pielover928** [ItsDeltin/Overwatch-Script-To-Workshop#58]
Steps to reproduce:
1. Declare a typed variable. Example:
     Color color;
2. Edit the code behind that line to include a statement. Example:
    This.number = 0;
    Color color;
3. The parser will crash.

**Comment by TrueCP6** [comment]
Crash log:
```
Unhandled Exception: System.ArgumentException: Requested value 'color' was not found.
   at System.Enum.TryParseEnum(Type enumType, String value, Boolean ignoreCase, EnumResult& parseResult)
   at System.Enum.Parse(Type enumType, String value, Boolean ignoreCase)
   at Deltin.Deltinteger.Parse.UseVarNode..ctor(UseVarContext context, BuildAstVisitor visitor) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\ast.cs:line 701
   at Deltin.Deltinteger.Parse.BuildAstVisitor.VisitUseVar(UseVarContext context) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\ast.cs:line 54
   at Deltin.Deltinteger.Parse.DefineNode..ctor(DefineContext context, BuildAstVisitor visitor) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\ast.cs:line 586
   at Deltin.Deltinteger.Parse.BuildAstVisitor.VisitDefine(DefineContext context) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\ast.cs:line 44
   at Deltin.Deltinteger.Parse.BuildAstVisitor.VisitBlock(BlockContext context) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\ast.cs:line 70
   at Deltin.Deltinteger.Parse.UserMethodNode..ctor(User_methodContext context, BuildAstVisitor visitor) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\ast.cs:line 853
   at Deltin.Deltinteger.Parse.BuildAstVisitor.VisitUser_method(User_methodContext context) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\ast.cs:line 59
   at Deltin.Deltinteger.Parse.RulesetNode..ctor(RulesetContext context, BuildAstVisitor visitor) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\ast.cs:line 550
   at Deltin.Deltinteger.Parse.BuildAstVisitor.VisitRuleset(RulesetContext context) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\ast.cs:line 24
   at Deltin.Deltinteger.Parse.ParsingData.GetRuleset(String file, String content) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\Parse.cs:line 126
   at Deltin.Deltinteger.Parse.ParsingData.GetObjects(String document, String file, TranslateRule globalTranslate, TranslateRule playerTranslate, Boolean isRoot) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\Parse.cs:line 147
   at Deltin.Deltinteger.Parse.ParsingData..ctor(String file, String content) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\Parse.cs:line 28
   at Deltin.Deltinteger.LanguageServer.Server.ParseDocument(String input) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Language Server\Server.cs:line 146
   at Deltin.Deltinteger.LanguageServer.Server.RequestLoop(Int32 serverPort) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Language Server\Server.cs:line 60
   at Deltin.Deltinteger.Program.Main(String[] args) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Program.cs:line 72
```

**Comment by ItsDeltin** [comment]
Fixed in v0.4.0.2, which is in v0.4's releases.
